examples/scripts/uptime.py

```diff
@@ -31,17 +31,26 @@ def v2_runner_on_failed(self, result, *args, **kwargs):
 
 def main():
     host_list = ['localhost', 'www.example.com', 'www.google.com']
-    Options = namedtuple('Options', ['connection', 'module_path', 'forks', 'remote_user',
-                                     'private_key_file', 'ssh_common_args', 'ssh_extra_args', 'sftp_extra_args',
-                                     'scp_extra_args', 'become', 'become_method', 'become_user', 'verbosity', 'check',
-                                     'diff'])
+    sources = ','.join(host_list)
+    # required for
+    # https://github.com/ansible/ansible/blob/devel/lib/ansible/inventory/manager.py#L204
+    if len(host_list) == 1:
+        sources += ','
```

Comment:
I'm ok with this change, however the additional formatting changes should be reverted.  The current formatting meets our code standards.